sdk/search/search-documents/test/public/utils/setup.ts

```diff
@@ -1,231 +1,306 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { env, isLiveMode, isPlaybackMode } from "@azure-tools/test-recorder";
+import { assertEnvironmentVariable, isLiveMode, isPlaybackMode } from "@azure-tools/test-recorder";
 import { OpenAIClient } from "@azure/openai";
 import { assert } from "chai";
 import {
   GeographyPoint,
   KnownAnalyzerNames,
   SearchClient,
+  SearchField,
   SearchIndex,
   SearchIndexClient,
   SearchIndexerClient,
+  VectorSearchAlgorithmConfiguration,
+  VectorSearchCompressionConfiguration,
+  VectorSearchVectorizer,
 } from "../../../src";
 import { delay } from "../../../src/serviceUtils";
 import { COMPRESSION_DISABLED } from "../../compressionDisabled";
 import { Hotel } from "./interfaces";
 
 export const WAIT_TIME = isPlaybackMode() ? 0 : 4000;
 
+function uniqueNameGenerator() {
```

Comment:
Would a `uuid` from @azure/core-util work here instead? Meaning deleting this helper and just using uuids for unique names